### DISCUSSION TITLE
Icon request: icon-NEW  / icon: "NEW"

### DISCUSSION THREAD

**Discussion by jomiguelff** [FortAwesome/Font-Awesome#21123] [Icon Requests]
How do you plan on using it?
We can use that the "New" icon for any new stuff you have in your site, or a new category.

It can be a icon like the certificate one: http://fontawesome.io/icon/certificate/  but with the "new" inside.

Something like this: 
![new](https://cloud.githubusercontent.com/assets/28120790/25508617/47fabf04-2b81-11e7-829c-6eaa1a1a2c68.gif)
![new-002](https://cloud.githubusercontent.com/assets/28120790/25508618/48120b0a-2b81-11e7-8c76-dc182c854912.jpg)

Thank you very much! This icon is going to be very useful!

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

#1331 was closed by the opener, so this request is not a duplicate

Please note that icons should be language independent, but this contain the word "new"

**Comment by SausageParty69** [comment]
+1

I guess the word "new" is worldwide? Even if not, I'd rather say most if not all people do understand the meaning..

**Comment by rjgotten** [comment]
-1

This can easily be built from layering the certificate icon behind some carefully CSS transform-ed text.
And _that_ will actually be localizable...

**Comment by Crissov** [comment]
Unicode emoji #222: U+1F195 🆕 `SQUARED NEW`

The [original Japanese designs](http://www.unicode.org/~scherer/emoji4unicode/snapshot/utc.html#e-B36) were not necessarily boring squares. 

| DoCoMo |	KDDI |	SoftBank |	Google |
|----|----|----|----|
| # 120 | # 334 | # 437 old# 198 | # B36
| 'New' ＮＥＷ | NEW | NEW | ---
| U+E6DD | U+E5B5 | U+E212 | U+FEB36
| SJIS-F982 | SJIS-F7E5 | SJIS-F7B2 | ---
| JIS-7867 | JIS-7867 | --- | ---
| ![](https://www.nttdocomo.co.jp/service/developer/make/content/pictograph/basic/images/120.gif) | ![](http://www001.upp.so-net.ne.jp/hdml/emoji/e/334.gif) ![](https://web-beta.archive.org/web/20071004235149im_/http://developer.openwave.com/documentation/xhtml_mp_css_reference/images/12-334.gif) | ![](http://creation.mb.softbank.jp/mc/tech/tech_pic/img/E212_20.gif) | ![](https://mail.google.com/mail/e/B36)

**Comment by supercreator** [comment]
+1 if the word is not good, it should have the other good design for the request. It’s needed.

**Comment by nachodd** [comment]
Maybe this little hack I made with CSS (like @rjgotten suggested) could be helpful for someone!

[https://codepen.io/nachodd/pen/VQrvjG](https://codepen.io/nachodd/pen/VQrvjG )

**Comment by tagliala** [comment]
@nachodd thanks

There is also an offical example on the docs with SVG: https://fontawesome.com/how-to-use/on-the-web/styling/layering

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Added to 7.2:
https://fontawesome.com/icons/burst-new?f=classic&s=solid
https://fontawesome.com/icons/rectangle-new?f=classic&s=solid

**Comment by jomiguelff** [comment]
Hahahaha i wrote this like 10 years ago

*Lic. José Miguel Fernández*

*Correo: ***@***.*** ***@***.***>*

**Comment by sensibleworld** [comment]
You know what they say! What's old is new again.
